tests/test_imports.py

```
@@ -0,0 +1,40 @@
+"""
+import pytest
+import os
+import sys
+
+# Add the project root to the Python path
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+def test_package_imports():
+    """Test that all main components can be imported from the package."""
+    try:
+        from lx_anonymizer.lx_anonymizer.report_reader import ReportReader
```

Ruff (0.8.2)

`lx_anonymizer.lx_anonymizer.report_reader.ReportReader` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)